ui/admin/app/routes/scopes/scope/roles/role/grants.js

```diff
@@ -21,7 +21,7 @@ export default class ScopesScopeRolesRoleGrantsRoute extends Route {
    */
   @action
   addGrant(role, grantString) {
-    role.grant_strings.unshiftObject(grantString);
+    role.grant_strings = [grantString].concat(role.grant_strings);
```

Comment:
Is there a reason you didn't use `unshift` now that's it a normal array? Does this still get tracked?

Alternatively this could have been:
```suggestion
    role.grant_strings = [grantString, ...role.grant_strings];
```

Reply:
I'll try it.  Changes to the _contents_ of the array are no longer tracked, which is why an assignment might be necessary.